### PR TITLE
PR: Split Explorer module

### PR DESCRIPTION
**Pull request by rlaverde**
Part of #4591 

Should this module be merged with project explorer? If yes, how?

I think It's better to have two modules, and project module depending of explorer module

### CODE REVIEW

**Comment by ccordoba12** [comment]
> I think It's better to have two modules, and project module depending of explorer module

Yes, I also it's better to have two separate plugins.

File: spyder/plugins/explorer/__init__.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+spyder.plugins.explorer
+===========
```
Comment (ccordoba12) [inline spyder/plugins/explorer/__init__.py]: Please extend these `==` to cover the entire line above.

**Comment by rlaverde** [comment]
I think this is ready :smile:

**Comment by ccordoba12** [comment]
> or should I merge working directory into it?

Nop. Let's leave things like that after we merge `split-plugins` with master.

**Pull request merged**